arcade/experimental/lights.py

```diff
@@ -15,7 +16,7 @@ def __init__(
         center_x: float,
         center_y: float,
         radius: float = 50.0,
-        color: Color = (255, 255, 255),
+        color: RGBOrA255 = WHITE,
```

Comment:
Should just be `RGBA`?

Reply:
To my understanding, passing to the GL shader discards the alpha channel:

```
        self._combine_program['ambient'] = ambient_color[:3]
```
I can make it RGBA if you'd like, but this is an advanced example. A nudge toward RGBA may annoy advanced users here.